### DISCUSSION TITLE
0004 XLS-4d Standard URI instructing a wallet to add a trustline

### DISCUSSION THREAD

**Discussion by RichardAH** [XRPLF/XRPL-Standards#25] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by RichardAH** [comment]
to be as backward compatible as possible with defacto existing implementations I suggest any instruction that doesn't explicitly have an action= parameter is default interpreted as a payment instruction. (That said: action=payment should probably be the canonical on XLS-2d)

**Comment by mvadari** [comment]
@RichardAH should this spec be moved into PR and kept in the repo or should it be closed? Note that this does not mean that the spec has been accepted/implemented/finalized.

I will move it to PR by copy-pasting the existing draft if there is no response in the next week.
